designs/charlie/src/front.mjs

```diff
@@ -99,6 +96,45 @@ function draftCharlieFront({
   points.flyCurveCp1 = points.flyBottom.shiftFractionTowards(points.flyCorner, options.flyCurve)
   points.flyCurveCp2 = points.flyCurveStart.shiftFractionTowards(points.flyCorner, options.flyCurve)
 
+  let topStitchDist = (1 - options.flyWidth) * 8
```

Comment:
Unless you plan to overwrite the variable later on, we prefer to use `const` in our codebase.
It's more declarative, somewhat more efficient, and reduces cognitive overhead when reading code.

Unless I missed something (I'm reading on my phone) this could be a `const`